reana/reana_dev/git.py

```diff
@@ -371,8 +372,30 @@ def git_clone(user, component, exclude_components):  # noqa: D301
     if exclude_components:
         exclude_components = exclude_components.split(",")
     components = select_components(component, exclude_components)
+
+    if "--target-directory" in sys.argv:
+        instalation_directory = os.path.realpath(target_directory + "/")
```

Comment:
```suggestion
        installation_directory = os.path.realpath(target_directory + "/")
```

Reply:
We can use `os.path.join` instead of `+`.
Just curious, is it strictly necessary to include the `"/"` at the end?

Reply:
I have just tested it and it seems to work without the `"/"`, so I just dropped it